vms/platformvm/client.go

```diff
@@ -120,6 +120,13 @@ type Client interface {
 		height uint64,
 		options ...rpc.Option,
 	) (map[ids.NodeID]*validators.GetValidatorOutput, error)
+	// GetProposedValidators returns the weights of the validator set of a provided
+	// subnet at the proposer's height.
```

Comment:
```suggestion
	// subnet at this node's current proposed height.
```

Technically the proposed height is determined on a per-node basis, and the node serving the RPC request likely isn't the same as the proposer of the next block.